sources/api/apiserver/src/server/mod.rs

```diff
@@ -543,6 +549,46 @@ async fn reboot() -> Result<HttpResponse> {
     Ok(HttpResponse::NoContent().finish())
 }
 
+/// Gets the set of report types supported by this host.
+async fn list_reports() -> Result<ReportListResponse> {
+    // Add each report to list response when adding a new handler
+    let data = vec![Report {
+        name: "cis".to_string(),
+        description: "CIS Bottlerocket Benchmark".to_string(),
+    }];
+    Ok(ReportListResponse(data))
+}
+
+/// Gets the Bottlerocket CIS benchmark report.
+async fn get_cis_report(query: web::Query<HashMap<String, String>>) -> Result<HttpResponse> {
+    let mut cmd = AsyncCommand::new("/usr/bin/bloodhound");
+
+    // Check for requested level, default is 1
+    if let Some(level) = query.get("level") {
+        cmd.arg("-l").arg(level);
+    }
+
+    // Check for requested format, default is text
```

Comment:
Does it make sense for us to specify the defaults to the underlying tool here? That way behavior changes to the tool's defaults don't impact the apiserver.

Reply:
I don't see ever changing defaults to `bloodhound`, so I tried to avoid sending unnecessary data here. Do you think it's important to make those defaults explicit here?

If we do update bloodhound, we can update the `apiserver` at the same time. Or is there a situation where we could end up with an older `apiserver` and newer `bloodhound`?

Reply:
I'm assuming since you approved with this, this was more of just a question versus a strong opinion. Since this has the approvals I am going to merge it, but please do open an issue (or just ping me) if you feel strongly that anything should change here. We still have some time to tweak things, especially something like this, so happy to follow up.